package.json

```diff
@@ -40,19 +41,19 @@
     "mochainon": "^2.0.0",
     "mockttp": "^0.9.1",
     "require-npm4-to-publish": "^1.0.0",
-    "resin-auth": "^2.0.0",
+    "balena-auth": "^3.0.0",
     "resin-config-karma": "^2.2.0",
```

Comment:
This?

Reply:
No - it's just an internal dev module, which isn't visible in the end product at all and won't need breaking changes to bump it here later, so it's not in my list for this week.

Reply:
It seems like it'd be easier to just manually `npm publish` the latest version to `balena-config-karma` (with no changes) and change this to `balena-config-karma: ^2.2.0` as part of this batch of updates, rather than having to do another set of PRs across all of our modules later just for the one module